config/locales/en.yml

```diff
@@ -115,6 +115,8 @@ en:
     referee_type: What kind of referee do you want to add?
     referee_destroy: Are you sure you want to delete this referee?
     referee_cancel: Are you sure you want to cancel this request for a reference?
+    decoupled_reference_cancel: Are you sure you want to cancel this reference request?
```

Comment:
Maybe call this `references_cancel_request` for consistency with the other recent additions. Means no need to revisit this when we do clean up further down the road.